eZ/Publish/API/Repository/LocationService.php

```diff
@@ -51,6 +51,19 @@ public function copySubtree(Location $subtree, Location $targetParentLocation);
      */
     public function loadLocation($locationId, array $prioritizedLanguages = null, bool $useAlwaysAvailable = null);
 
+    /**
+     * Loads several location objects from its $locationIds.
+     *
+     * Returned list of Locations will be filtered by what is found and what current user has access to.
+     *
+     * @param array $locationIds
+     * @param string[]|null $prioritizedLanguages Filter on and use as prioritized language code on translated properties of returned objects.
+     * @param bool|null $useAlwaysAvailable Respect always available flag on content when filtering on $prioritizedLanguages.
+     *
+     * @return \eZ\Publish\API\Repository\Values\Content\Location[]|iterable
+     */
+    public function loadLocationList(array $locationIds, array $prioritizedLanguages = null, bool $useAlwaysAvailable = null): iterable;
```

Comment:
The `loadLocationList` method is missing on `\eZ\Publish\Core\REST\Client\LocationService`

Reply:
fixed in 6cb281cc3dc08714a0b309cd20865165e2ca3122